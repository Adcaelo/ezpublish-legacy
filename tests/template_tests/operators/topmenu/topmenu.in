{var $a = topmenu()}
{$a[0]["url"]}
