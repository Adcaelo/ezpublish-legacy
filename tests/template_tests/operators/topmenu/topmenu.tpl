{let a=topmenu()}
{$a.0.url}

{/let}
