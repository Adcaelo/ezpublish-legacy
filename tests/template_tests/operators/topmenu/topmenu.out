content/view/full/2
